### PR TITLE
Pulse Connect Secure VPN Bugbash Changes

### PR DESCRIPTION
**Pull request by aprakash13**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Detections/PulseConnectSecure/PulseConnectSecureVPN-BruteForce.yaml [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-  let timeframe = ago(1h);
-  let threshold = 20;
-  PulseConnectSecure
-  | where TimeGenerated >= timeframe
-  | where Messages contains "Login failed"
-  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), count() by User, Source_IP
-  | where count_ > threshold
-  | extend timestamp = StartTime, AccountCustomEntity = User, IPCustomEntity = Source_IP
+id: 34663177-8abf-4db1-b0a4-5683ab273f44
+name: Potential Brute Force Attempts
```
Comment (shainw) [inline Detections/PulseConnectSecure/PulseConnectSecureVPN-BruteForce.yaml:2]: This rename makes more sense than the other, as the initial has BF identified.  Your name is a generic name, probably should indicate it came from PulseConnectSecure with a name such as - `PulseConnectSecure - Potential Brute Force Attempts`

File: Detections/PulseConnectSecure/PulseConnectSecureVPN-PasswordSpray.yaml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 id: 1fa1528e-f746-4794-8a41-14827f4cb798
-name: Large Number of Distinct Failed User Logins
+name: Potential Password Spray Attempts
```
Comment (shainw) [inline Detections/PulseConnectSecure/PulseConnectSecureVPN-PasswordSpray.yaml:2]: I would be careful with this rename, we don't know what Pulse Secure is identifying here.  We should add - `PulseConnectSecure - ` to the beginning though.

File: Detections/PulseConnectSecure/PulseConnectSecureVPN-BruteForce.yaml (hunk 1 above)
Comment (aprakash13) [reply Detections/PulseConnectSecure/PulseConnectSecureVPN-BruteForce.yaml]: Thanks Shain. Made the suggested changes.

File: Detections/PulseConnectSecure/PulseConnectSecureVPN-PasswordSpray.yaml (hunk 2 above)
Comment (aprakash13) [reply Detections/PulseConnectSecure/PulseConnectSecureVPN-PasswordSpray.yaml]: Thanks Shain. Renamed it and also added PulseConnectSecure in the begining of the name to identify Data Source.

**Review by shainw — APPROVED** [review]

**Pull request merged**